CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# Release v1.4 (2018-07-12)
+
+## New Features
+  * Support for GRPC unix domain socket type. Socket types tcp, tcp4, tcp6, unix 
+  and unixpacket can be used with GRPC. Desired socket type and address/file can be 
+  specified via grpc configuration file ([example here](rpc/grpc/grpc.conf)). More
+  information in the [readme](rpc/grpc/README.md)
+  * Rest plugin security improved. Security features are usage of client/server certificates
+  (HTTPS) and basic HTTP authentication with username and password. Those features 
+  are disabled by default. Information about how to use it and example can be found in the 
+  [readme](rpc/rest/README.md)
+  
+## Other
+  * Example configuration files with description were added to every plugin which 
+  supports/uses them. 
+  * Package **etcdv3** was renamed to **etcd**
```

Comment:
Please put this at the top as *Breaking Changes* and also mention that it affects flags and configuration file as well.